app/controllers/organizations_controller.rb

```diff
@@ -56,12 +56,21 @@ def show_groupings
   def update
     result = Organization::Editor.perform(organization: current_organization, options: update_organization_params.to_h)
 
-    if result.success?
-      flash[:success] = "Successfully updated \"#{current_organization.title}\"!"
-      redirect_to current_organization
-    else
-      current_organization.reload
-      render :edit
+    respond_to do |format|
+      format.html do
+        if result.success?
```

Comment:
I didn't change any of the current functionality of the html action. I just moved it up into the `respond_to` section for html.